lib/utils.js

```diff
@@ -5,19 +5,38 @@ const exec = require('child_process').exec;
 const qs = require('qs');
 const crypto = require('crypto');
 
-const hasOwn = {}.hasOwnProperty;
-const isPlainObject = require('lodash.isplainobject');
+const hasOwn = Function.bind.call(
+  Function.call,
+  Object.prototype.hasOwnProperty
+);
 
-const OPTIONS_KEYS = [
+const OPTIONS_KEYS = new Set([
   'api_key',
   'idempotency_key',
   'stripe_account',
   'stripe_version',
-];
+]);
 
 const utils = (module.exports = {
-  isOptionsHash: (o) => {
-    return isPlainObject(o) && OPTIONS_KEYS.some((key) => hasOwn.call(o, key));
+  isSerializableObject(obj) {
+    if (typeof obj !== 'object') {
+      return false;
+    }
+    try {
+      const str = JSON.stringify(obj);
+      return str === JSON.stringify(JSON.parse(str));
+    } catch (_) {
+      return false;
+    }
+  },
+
+  isOptionsHash(o) {
+    return (
+      this.isSerializableObject(o) &&
```

Comment:
I actually think we don't need to be this careful – it's fine if people pass us a class instance with the right properties, or whatever. 

I think we can get away with this:

```
isOptionsHash(o) {
  return o && typeof o === 'object' && OPTIONS_KEYS.some(key => hasOwn(o, key))
}
```
(accordingly, probably best to put `OPTIONS_KEYS` back to an array, since it's better to iterate over a known-short array than a possibly-long one).